exasol_advanced_analytics_framework/interface/create_event_handler_udf.py

```diff
@@ -1,10 +1,93 @@
+import importlib
+from collections import OrderedDict
+from pathlib import PurePosixPath
+from exasol_bucketfs_utils_python.bucketfs_factory import BucketFSFactory
+from exasol_bucketfs_utils_python.bucketfs_location import BucketFSLocation
+from exasol_advanced_analytics_framework.event_handler.event_handler_base \
+    import EventHandlerBase
+from exasol_advanced_analytics_framework.event_handler.event_handler_context \
+    import EventHandlerContext
+from exasol_advanced_analytics_framework.event_handler.event_handler_result \
+    import EventHandlerResult
+from exasol_advanced_analytics_framework.event_handler.event_handler_state \
+    import EventHandlerState
+from exasol_advanced_analytics_framework.context_wrapper.udf_context_wrapper \
+    import UDFContextWrapper
+
 
 class CreateEventHandlerUDF:
+
     def __init__(self, exa):
         self.exa = exa
 
     def run(self, ctx) -> None:
-        ctx.emit("return query")
-        ctx.emit("status")
-        ctx.emit("query 1")
-        ctx.emit("query 2")
+        # get and set method parameters
+        parameters = ctx.parameters
+        bucketfs_connection = ctx.bucketfs_connection
+        bucketfs_location = BucketFSFactory().create_bucketfs_location(
+            url=bucketfs_connection.address,
+            user=bucketfs_connection.user,
+            pwd=bucketfs_connection.password,
+            base_path=parameters["base"]
+        )
+        bucketfs_path = PurePosixPath(
```

Comment:
I think, we don't need this, we can manage everything via the bucketfs_connection and the step parameter